content/tr/event-streaming.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Olay Akışı
+status: Completed
+category: consept
+tags: ["yöntem", "ağ tanımlama", ""]
+---
+
+Olay akışı, yazılımın, yaptıklarını sürekli olarak iletmek için olay verilerini bir uygulamadan diğerine gönderdiği bir yaklaşımdır. 
+Yaptığı her şeyi diğer tüm servislere yayınlayan bir servis düşünün. 
+Bir servis tarafından gerçekleştirilen her faaliyet bir olay olarak adlandırılır, dolayısıyla olay akışı olarak adlandırılır. 
+Örneğin, NASDAQ her saniye hisse senedi ve ticari malların fiyatlandırılması hakkında güncellemeler alır. 
+Belirli bir stok setini takip eden bir uygulamanız olsaydı, bu bilgiyi neredeyse gerçek zamanlı olarak almak isterdiniz. 
+Yahoo! Finans, NASDAQ'tan çekilen ve uygulamalarındaki bilgileri (veya olayları) ona abone olan herhangi bir uygulamaya gönderen (veya akıtan) bir [API](/application-programming-interface/) sağlar. 
```

Comment:
Noticed something as well. This URL must include the `/tr/` prefix to work. Can you please update the link @yasin-herken?

Reply:
It is ready now.

Reply:
Thank you @yasin-herken.